collectors/bzimport/tests/test_convertors.py

```diff
@@ -1451,6 +1460,53 @@ def test_groups(self, request, bz_groups, ldap_read_group, ldap_write_group):
         assert flaw.acl_read == ldap_read_group
         assert flaw.acl_write == ldap_write_group
 
+    @pytest.mark.parametrize(
+        "hightouch,hightouch_lite,creation_date",
+        [
+            (None, None, None),  # no flags
+            ("-", "-", None),  # rejected
+            ("+", "", "2021-09-15T08:19:46Z"),  # approved
+        ],
+    )
+    def test_major_incident_start_dt(self, hightouch, hightouch_lite, creation_date):
+        """
+        Test that if the appropriate flags are set to be a MI, then the MI start
+        date is properly set from the flag creation date.
+        """
+        flaw_bug = self.get_flaw_bug()
+        flaw_bug["flags"] = []
+
+        if hightouch is not None and hightouch_lite is not None:
```

Comment:
Maybe a stupid question, but why both `hightouch` and `hightouch_lite` must be `not None`?

Reply:
These flags are optional in Bugzilla. I'm testing the case in which these flags aren't present, in which case the date shouldn't be set either (as it would have a major incident state of `NOVALUE`)